docs/2023/Health Claims Exchange/Build Javascript SDK for HCX Participant System to help in integrating with HCX/updates/2023-08-04.md

```diff
@@ -4,13 +4,17 @@ author: Mohit Kumar
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [x] Complete the Outgoing Requests.
+- [x] Complete the Incoming Requests.
 
 ## Screenshots / Videos 
 
 ## Contributions
+The links of commitss are given:
+- [Commit1](https://github.com/Swasth-Digital-Health-Foundation/integration-sdks/pull/64/commits/b809482721e371091ee36d5723c9a9df8cb71a76)
+- [Commit2](https://github.com/Swasth-Digital-Health-Foundation/integration-sdks/pull/64/commits/905e88ed499b70d7b9f46c5895c8af9f30a80073)
+- [Pull Request](https://github.com/Swasth-Digital-Health-Foundation/integration-sdks/pull/64)
```

Comment:
Hey you have attached the commits of your linked pull request, please remove commits link and just attach a PR link